### ISSUE TITLE
Improve design for groups so they handle groups within groups better

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#202]
## Description
Our design for our repeating groups do not handle groups in groups very well. We need a solution where the app developer can add multiple groups within groups without breaking the design. If there is no quick fix for the current design, we need to either redo the design of our groups to better manage groups in groups or make an "advanced" version for more complex tables and groups.

## Screenshots
> Screenshots or links to Figma (make sure your sketch is public)

## Considerations
- How many layers of group should be allowed? 
- Consider smaller screens

## Acceptance criteria
> Describe criteria here (i.e. What is allowed/not allowed (negative tesing), validations, error messages and warnings etc.)

## Specification tasks

- [ ] Development tasks are defined
- [ ] Test design / decide test need

## Development tasks
> Add tasks here

## Definition of done
Verify that this issue meets [DoD](https://confluence.brreg.no/display/T3KP/Definition+of+Done#DefinitionofDone-DoD%E2%80%93utvikling) (Only for project members) before closing.

- [ ] Documentation is updated (if relevant)
  - [ ] Technical documentation (docs.altinn.studio)
  - [ ] User documentation (altinn.github.io/docs)
- [ ] QA
- [ ] Manual test is complete (if relevant)
- [ ] Automated test is implemented (if relevant)
- [ ] All tasks in this userstory are closed (i.e. remaining tasks are moved to other user stories or marked obsolete)

**Comment by bjosttveit** [comment]
Is this issue still relevant or can it be closed?

<img width="1553" alt="image" src="https://github.com/Altinn/app-frontend-react/assets/47412359/dae6ef37-1017-4ccb-8385-a02bfd164075">

**Comment by altinnadmin** [comment]
> Is this issue still relevant or can it be closed?

@bjosttveit Nice, looks promising :)

Is "repeating groups within repeating groups" documented somewhere? Limitations?

**Comment by bjosttveit** [comment]
Could not find anything specifically about this in the docs. One limitation which has been there since the beginning is maximum 2 "levels" of nested repeating groups. I just figured we could maybe close this issue since the design has changed many times since this issue was created and may not be relevant anymore.

**Comment by altinnadmin** [comment]
Yeah, not even sure how this look "way back then" when @Febakke registered this issue almost 3 years ago, since no screenshot was provided.
